tool/tsh/tctl_test.go

```diff
@@ -94,3 +96,81 @@ func TestLoadConfigFromProfile(t *testing.T) {
 		})
 	}
 }
+
+func TestSetAuthServerFlagWhileLoggedIn(t *testing.T) {
+	tmpHomePath := filepath.Clean(t.TempDir())
```

Comment:
I'm curious why we need `filepath.Clean`, or is it just because?

Reply:
It's just because